guiEditor/src/components/log/logComponent.tsx

```diff
@@ -0,0 +1,64 @@
+

```

Comment:
This should be shared with nme

Reply:
Not quite sure how to move this since it depends on global state.

Reply:
Going to fix this later in the week with another PR when I move to share components. I have an idea how I'm going to remove global state from this and other components with the same dependency.